src/renderer/saves/RecentSaves.tsx

```diff
@@ -126,6 +103,29 @@ export default function RecentSaves(props: SaveFileSelectorProps) {
       renderValue: (save) => formatTime(save.lastModified),
       sortFunction: numericSorter((val) => val.lastModified),
     },
+    {
+      key: 'remove',
+      name: '',
+      width: 40,
+      renderCell: (params) => (
+        <button
+          style={{
+            padding: 0,
+            display: 'grid',
+            marginLeft: 'auto',
+            marginTop: 'auto',
+            marginBottom: 'auto',
+            // backgroundColor: '#990000',
```

Comment:
Can we leave this button red? That way it looks more like a "remove" button

Reply:
Yeah for sure. I like the white background better but that can just be a local change